examples/docs_beta_snippets/docs_beta_snippets/guides/components/shell-script-component/2-shell-command-empty.py

```diff
@@ -1,5 +1,4 @@
-import dagster as dg
-
+from dagster import Definitions
```

Comment:
This file is automatically generated, so we'd have to use `import dagster as dg` in that script. Will make a note of that.